pages/myforms/[[...path]].tsx

```diff
@@ -140,14 +140,16 @@ export const getServerSideProps = requireAuthentication(
       const templates = (await getAllTemplates(ability, id)).map((template) => {
         const {
           id,
-          form: { titleEn, titleFr },
+          form: { titleEn = "", titleFr = "" },
```

Comment:
Fixes an issue when "no templates exist" but we're trying to destructure

<img width="1005" alt="Screen Shot 2023-01-31 at 12 26 01 PM" src="https://user-images.githubusercontent.com/62242/215851826-dcd8b066-0028-4fe4-91bd-9e94e32fbb01.png">